packages/api/index.ts

```diff
@@ -295,7 +295,7 @@ export interface BuiltInLLMParams {
 
 export interface BuiltInLLMState<T> {
   pending: boolean;
-  result?: T;
+  result?: BuiltInLLMContent;
```

Comment:
This subverts the intent for this to be a generic type, but we must return `BuiltInLLMContent` from anything other than `generateObject` calls.